qiskit/algorithms/state_fidelities/base_state_fidelity.py

```diff
@@ -0,0 +1,305 @@
+                    parametrized_circuit_1, parametrized_circuit_2
+                )
+                circuits.append(circuit)
+                # update cache
+                self._circuit_cache[id(circuit_1), id(circuit_2)] = circuit
+
+        return circuits
+
+    def _construct_value_list(
+        self,
+        circuits_1: Sequence[QuantumCircuit],
+        circuits_2: Sequence[QuantumCircuit],
+        values_1: Sequence[float] | Sequence[Sequence[float]] | None = None,
+        values_2: Sequence[float] | Sequence[Sequence[float]] | None = None,
+    ) -> list[float]:
+        """
+        Preprocesses input parameter values to match the fidelity
+        circuit parametrization, and return in list format.
+
+        Args:
+           circuits_1: (Parametrized) quantum circuits preparing the
+                        first list of quantum states.
+           circuits_2: (Parametrized) quantum circuits preparing the
+                        second list of quantum states.
+           values_1: Numerical parameters to be bound to the first circuits.
+           values_2: Numerical parameters to be bound to the second circuits.
+
+        Returns:
+             List of parameter values for fidelity circuit.
+
+        """
+        values_1 = self._preprocess_values(circuits_1, values_1)
+        values_2 = self._preprocess_values(circuits_2, values_2)
+
+        values = []
+        if len(values_2[0]) == 0:
+            values = list(values_1)
+        elif len(values_1[0]) == 0:
+            values = list(values_2)
+        else:
+            for (val_1, val_2) in zip(values_1, values_2):
+                values.append(val_1 + val_2)
+
+        return values
+
+    @abstractmethod
+    def _run(
+        self,
+        circuits_1: QuantumCircuit | Sequence[QuantumCircuit],
+        circuits_2: QuantumCircuit | Sequence[QuantumCircuit],
+        values_1: Sequence[float] | Sequence[Sequence[float]] | None = None,
+        values_2: Sequence[float] | Sequence[Sequence[float]] | None = None,
+        **run_options,
+    ) -> StateFidelityResult:
+        r"""
+        Computes the state overlap (fidelity) calculation between two
+        (parametrized) circuits (first and second) for a specific set of parameter
+        values (first and second).
+
+        Args:
+            circuits_1: (Parametrized) quantum circuits preparing :math:`|\psi\rangle`.
+            circuits_2: (Parametrized) quantum circuits preparing :math:`|\phi\rangle`.
+            values_1: Numerical parameters to be bound to the first set of circuits
+            values_2: Numerical parameters to be bound to the second set of circuits.
+            run_options: Backend runtime options used for circuit execution. The order
+            of priority is: run_options in ``run`` method > fidelity's default
+            run_options > primitive's default setting.
+            Higher priority setting overrides lower priority setting.
```

Comment:
```suggestion
            run_options: Backend runtime options used for circuit execution. The order
                of priority is\: run_options in ``run`` method > fidelity's default
                run_options > primitive's default setting.
                Higher priority setting overrides lower priority setting.
```
The continuation text needs indenting, otherwise it comes out incorrect. I think the colon, needs escaping too.

![image](https://user-images.githubusercontent.com/40241007/188218639-7bed31d1-79f8-46dc-b18e-0282e9d0343d.png)